src/components/global/PlaygroundTabs/index.tsx

```diff
@@ -7,7 +7,8 @@
 
 import React, { useState, cloneElement, isValidElement, type ReactElement, useRef, createRef, useEffect } from 'react';
 import useIsBrowser from '@docusaurus/useIsBrowser';
-import { useScrollPositionBlocker, duplicates } from '@docusaurus/theme-common';
+import { duplicates } from '@docusaurus/theme-common';
+import { useScrollPositionBlocker } from '@docusaurus/theme-common/internal';
```

Comment:
`useScrollPositionBlocker` is needed to [maintain page position](https://github.com/facebook/docusaurus/pull/5618) for clicked grouped tabs. However, Docusaurus [does not encourage](https://docusaurus.io/community/release-process#theming-public-api) for users to import anything from ` '@docusaurus/theme-common/internal'` unless it's a swizzled component.

I propose to leave this for now for the following reasons:
Keep in mind PlaygroundTabs is a swizzled component from Docusaurus's Tabs.

The PlaygroundTabs currently fails web accessibility because it has buttons ([left arrow](https://github.com/ionic-team/ionic-docs/blob/71e57ed60f06be63c883963deb54af0c751278ae/src/components/global/PlaygroundTabs/index.tsx#L143-L166) and [right arrow](https://github.com/ionic-team/ionic-docs/blob/71e57ed60f06be63c883963deb54af0c751278ae/src/components/global/PlaygroundTabs/index.tsx#L190-L213)) inside a list (`<ul>`). A list can only have `li` as their children. In order to fix this accessibility issue, we should try to find a way to move the buttons outside of the tabs. If this is possible, then we won't need to use Docusaurus' Tabs logic and we can replace it with:

```tsx
// src/components/global/PlaygroundTabs/index.tsx
<div>
   <LeftButton />
   <Tabs className="unique-tabs">
     <TabItem value="X">src/app/example.component.ts</TabItem>
     <TabItem value="X">src/app/example.component.html</TabItem>
     <TabItem value="X">src/app/example.component.css</TabItem>
   </Tabs>
   <RightArrow />
<div>
``` 

Thoughts? @liamdebeasi @sean-perkins

Reply:
Can we make a bug ticket with this information (include what the desired behavior is and ideas for how to implement)? We can prioritize it so it gets fixed quickly.

Reply:
Created the ticket. 

Adding the following comment for records:
I do want to point out that my proposed solution might not completely remove the need to use `useScrollPositionBlocker` if this [PR](https://github.com/ionic-team/ionic-docs/pull/3169/files#diff-448a7530c1f7d78c814f204783575fb780358f357f6eee5a69f50ff4b7f0f0b6R16) is approved.

Reply:
Resolving since we determined to continue using it until an issue arises.